### PR TITLE
Problemdetailsmiddleware

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
Fix for this issue https://github.com/dotnet/AspNetCore.Docs.Samples/issues/60 @Rick-Anderson

### CODE REVIEW

File: fundamentals/middleware/problem-details-service/Program.cs [hunk 1]
```diff
@@ -50,36 +51,9 @@
...
-    if (radicand < 0)
-    {
-        var errorType = new MathErrorFeature { MathError =
-                                               MathErrorType.NegativeRadicandError };
-        context.Features.Set(errorType);
-        return Results.BadRequest();
-    }
-
-    return Results.Ok(Math.Sqrt(radicand));
-});
```
Comment (Rick-Anderson) [inline fundamentals/middleware/problem-details-service/Program.cs:80]: Why remove these endpoints?

**Comment by Rick-Anderson** [comment]
@sammychinedu2ky  normal work flow is after a PR is merged you:

```
git checkout main
git pull origin main
git checkout -b MyNewBranch
```

Then you get a cleaner PR without all the old commits.

**Comment by sammychinedu2ky** [comment]
> @sammychinedu2ky normal work flow is after a PR is merged you:
> 
> ```
> git checkout main
> git pull origin main
> git checkout -b MyNewBranch
> ```
> 
> Then you get a cleaner PR without all the old commits.

alright thanks. sorry about that was a mistake from my end

**Comment by sammychinedu2ky** [comment]
I have reverted it @Rick-Anderson

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
Thanks for the quick solution and code.

**Comment by sammychinedu2ky** [comment]
My pleasure. Thanks @Rick-Anderson

**Pull request merged**